Sources/ProjectDescription/Headers.swift

```diff
@@ -82,4 +88,48 @@ public struct Headers: Codable, Equatable {
             exclusionRule: exclusionRule
         )
     }
+
+    public static func headers(from list: FileList,
```

Comment:
I would make this private as we have already the other 2 to define the Bool value

Reply:
done